Mathlib/Data/Complex/Basic.lean

```diff
@@ -628,6 +628,22 @@ def ofReal : ℝ →+* ℂ where
 theorem ofReal_eq_coe (r : ℝ) : ofReal r = r :=
   rfl
 
+variable {α : Type*}
+
+@[simp] lemma ofReal_comp_add (f g : α → ℝ) : ofReal' ∘ (f + g) = ofReal' ∘ f + ofReal' ∘ g :=
```

Comment:
```suggestion
@[simp] lemma ofReal'_comp_add (f g : α → ℝ) : ofReal' ∘ (f + g) = ofReal' ∘ f + ofReal' ∘ g :=
```
etc

Reply:
Though I think `ofReal` is intended as an implementation detail?

Reply:
Your suggestion goes against the local naming convention. I am happy to change the local naming convention, but it's scope-creep for this PR

Reply:
Ah, I hadn't noticed that.

Maybe it would be better to write this with `(\u)`  notation?

Reply:
Yeah but then I need to type-ascript everything, which I thought wasn't really worth it when I can call the function directly

Reply:
Well, you should only need to do one side, but point taken nevertheless.